roles/dns_adblocking/tasks/main.yml

```diff
@@ -27,6 +27,13 @@
       notify:
         - restart dnsmasq
 
+    - name: VPN hosts file configured
```

Comment:
Looks like a dirty solution. I believe it might be implemented by a native way. eg: Delegate DHCP to dnsmasq instead of StrongSwan

Reply:
I wanted to do that, but I couldn't figure out a way for dnsmasq to know how to assign it a hostname based on the Algo user's name. Is there a way to do that that? If so, then I agree that would be much better.